pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/clusterconfig/index.md

```diff
@@ -56,6 +56,8 @@ enterprise: false
         spec:
           ami:
             id: ami-0d1ab6c50a5f30339
+          cloudInit:
+            insecureSkipSecretsManager: false
```

Comment:
oops sorry this needs to be true not false https://cluster-api-aws.sigs.k8s.io/crd/index.html#infrastructure.cluster.x-k8s.io/v1alpha4.CloudInit

i misread the API docs